### PR TITLE
hvplot 0.8.1

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**Jira ticket:** [PKG-731](https://anaconda.atlassian.net/browse/PKG-731) (hvplot-0.8.1)

**The upstream data:**
Github releases:  https://github.com/holoviz/hvplot/releases
[Diff between the latest and previous upstream releases](https://github.com/holoviz/hvplot/compare/0.8.0...0.8.1)
License: https://github.com/holoviz/hvplot/blob/master/LICENSE
Changelog: https://github.com/holoviz/hvplot/blob/master/CHANGELOG.md
Requirements:
 * setup.py:  https://github.com/holoviz/hvplot/blob/v0.8.1/setup.py
 *  pyproject.toml:  https://github.com/holoviz/hvplot/blob/v0.8.1/pyproject.toml

**_Actions:_**

1. Skip `s390x`
2. Remove pinnings from `host`
3. Update `run` pinnings
4. Add a `description`

**Package's statistics**
<details>

 * Priority B | effort: easy | Category: anaconda_affiliated | subcategory:  | pkg_type: python
 * Outdated platfroms: 'linux-64', 'linux-aarch64', 'linux-ppc64le', 'osx-64', 'osx-arm64', 'win-64'
 * Latest version: 0.8.1
 * Release on PyPi:
    * version: 0.8.1
    * date: 2022-08-26T12:37:12
* [PyPi history](https://pypi.org/project/hvplot/#history)
 * Popularity: 
    * 3 months downloads: 308625.0
    * All time:  261677 downloads -  hvplot  0.8.1  A high-level plotting API for the PyData ecosystem built on HoloViews  

</details>

**Other checks:**

<details>

1. - [x] https://github.com/holoviz/hvplot/tree/v0.8.1 exists
5. - [x] https://hvplot.holoviz.org is present
6. - [x] Check the pinnings
7. - [x] https://github.com/holoviz/hvplot/blob/master/CHANGELOG.md exists
8. - [x] Additional research
    https://github.com/holoviz/hvplot/issues
    200
9. - [x] `dev_url` is present
    https://github.com/holoviz/hvplot
10. - [x] `doc_url` is present
    https://hvplot.holoviz.org
11. - [x] Verify that the `build_number` is correct
12. - [x] has `setuptools`
13. - [x] has `wheel`
14. - [x] `pip` in test
15. - [x] Verify the test section
16. - [ x Verify if the package is `architecture specific`
17. - [x] Verify that private modules are not mentioned in the recipe For example: (_private_module)
18.  - [x] license_file: LICENSE is present
19. - [x] License: BSD-3-Clause
    - [x] License is `spdx` compliant
 * Check if the license identifier has correct name from the SPDX License List

| Identifier                           |
|:-------------------------------------|
| BSD-3-Clause                         |
| BSD-3-Clause-Attribution             |
| BSD-3-Clause-Clear                   |
| BSD-3-Clause-LBNL                    |
| BSD-3-Clause-Modification            |
| BSD-3-Clause-No-Military-License     |
| BSD-3-Clause-No-Nuclear-License      |
| BSD-3-Clause-No-Nuclear-License-2014 |
| BSD-3-Clause-No-Nuclear-Warranty     |
| BSD-3-Clause-Open-MPI                |

20. - [x] license_family BSD is present
</details>

**Links:**
* [Anaconda Recipes feedstock](https://github.com/AnacondaRecipes/hvplot-feedstock)
* [Update branch](https://github.com/AnacondaRecipes/hvplot-feedstock/tree/0.8.1)
* [conda-forge recipe](https://github.com/{upstream}/{feedstock_name}-feedstock)
* [PyPI](https://pypi.org/project/hvplot)
* [Diff between upstream and feature branch](https://github.com/conda-forge/hvplot-feedstock/compare/main...AnacondaRecipes:0.8.1)
* [Diff between origin and feature branch](https://github.com/AnacondaRecipes/hvplot-feedstock/compare/master...AnacondaRecipes:0.8.1)
* [The last merged PRs](https://github.com/AnacondaRecipes/hvplot-feedstock/pulls?q=is%3Apr+is%3Amerged+sort%3Aupdated-desc+)

**Updating the recipe:**
If the recipe needs additional modification the update branch can be modified. Note that the PR diffs are not updated with these changes.
```
git clone -b 0.8.1 git@github.com:AnacondaRecipes/hvplot-feedstock.git
```

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,32 +1,34 @@
...
   number: 0
+  # panel >=0.13.0 and nodejs aren't available on s390x
+  skip: True # [py<36 or s390x]
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - python
-    - pyct >=0.4.4
-    - param >=1.12.0
```
Comment (cbouss) [inline recipe/meta.yaml:20]: The host pinning should still match what is set here:
https://github.com/holoviz/hvplot/blob/v0.8.1/pyproject.toml

The point brought up yesterday does not apply here as neither pyct and param are run dependencies, and neither are set in conda_build_config.yaml.
Besides, removing the lower bound without setting an explicit version is equivalent to setting >=0.

**Review by cbouss — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:20]: Thanks, I've updated it. 
Our docs about using pinnings in `host` don't cover all use cases and I'm still a bit confused

Comment (cbouss) [reply recipe/meta.yaml:20]: I agree, pinning right is too complicated at the moment.

**Review by cbouss — APPROVED** [review]
LGTM

**Review by varlackc — COMMENTED** [review]
Not sure if we are still want to add pinnings in the host section as we used to in the past.
https://github.com/AnacondaRecipes/hvplot-feedstock/blob/06e8d9f3c435dd2165ae6cf28fcc01e3587451ea/recipe/meta.yaml#L21&L24

This is a point open to debate, while we no longer support versions older than `python 3.7` there is an argument to be made on keeping some dependencies as close as possible to the upsteram. In that context it might be a good idea to update the `skip section` to reflect the following `python` requirement in the upstream.
https://github.com/holoviz/hvplot/blob/v0.8.1/setup.py#L137

 python_requires=">=3.6"

**Comment by skupr-anaconda** [comment]
> Not sure if we are still want to add pinnings in the host section as we used to in the past. https://github.com/AnacondaRecipes/hvplot-feedstock/blob/06e8d9f3c435dd2165ae6cf28fcc01e3587451ea/recipe/meta.yaml#L21&L24
> 
> This is a point open to debate, while we no longer support versions older than `python 3.7` there is an argument to be made on keeping some dependencies as close as possible to the upsteram. In that context it might be a good idea to update the `skip section` to reflect the following `python` requirement in the upstream. https://github.com/holoviz/hvplot/blob/v0.8.1/setup.py#L137
> 
> python_requires=">=3.6"

It's already `py<36`
https://github.com/AnacondaRecipes/hvplot-feedstock/pull/4/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR15

**Pull request merged**